jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/Lambda.kt

```diff
@@ -22,6 +31,25 @@ object Lambda {
     }
 }
 
+// @see https://docs.aws.amazon.com/lambda/latest/dg/limits.html
+object LambdaLimits {
+    const val MIN_MEMORY = 128
+    const val MAX_MEMORY = 3008
+    const val MEMORY_INCREMENT = 64
+    const val DEFAULT_MEMORY_SIZE = 128
+    const val MIN_TIMEOUT = 1
+    @JvmField val MAX_TIMEOUT = TimeUnit.MINUTES.toSeconds(15).toInt()
+    @JvmField val DEFAULT_TIMEOUT = TimeUnit.MINUTES.toSeconds(5).toInt()
+}
+
+object LambdaWidgets {
+    fun lambdaTimeout(): SliderPanel =
```

Comment:
If you annotate these methods with `@JvmStatic` you won't have to grab the `INSTANCE` in the java code they'll appear as top-level static methods.

Reply:
Done.